### PR TITLE
V18.x update seeds issue#4949

### LINKED ISSUE #4949: Update hardcoded seeds.
see: https://github.com/dashpay/dash/tree/develop/contrib/seeds

### PR DESCRIPTION
**Pull request by knst**
Mainnet: 1716101
    Testnet: 774784
Closes #4949

### CODE REVIEW

**Comment by PastaPastaPasta** [comment]
This should be merged to develop, then backported

**Review by PastaPastaPasta — APPROVED** [review]
utACK for squash merge

File: contrib/seeds/README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Utility to generate the seeds.txt list that is compiled into the client
 
 The seeds compiled into the release are created from the current protx list, like this:
 
-    dash-cli protx list valid 1 1185193 > protx_list.json
+    dash-cli protx list valid 1 1716101 > protx_list.json
```
Comment (UdjinM6) [inline contrib/seeds/README.md:8]: nit: no need to update docs, this is just an example

**Review by UdjinM6 — APPROVED** [review]
utACK

**Pull request merged**